src/Libraries/RevitNodes/Elements/InternalUtilities/ElementQueries.cs

```diff
@@ -65,19 +64,13 @@ public static IList<Element> OfFamilyType(FamilyType familyType)
         {
             if (familyType == null) return null;
 
-            var instanceFilter = new Autodesk.Revit.DB.ElementClassFilter(typeof(Autodesk.Revit.DB.FamilyInstance));
-            var fec = new Autodesk.Revit.DB.FilteredElementCollector(DocumentManager.Instance.CurrentDBDocument);
+            var instanceFilter = new DB.ElementClassFilter(typeof(DB.FamilyInstance));
+            var fec = new DB.FilteredElementCollector(DocumentManager.Instance.CurrentDBDocument);
 
-            var familyInstances = fec.WherePasses(instanceFilter)
+            return fec.WherePasses(instanceFilter)
                 .WhereElementIsNotElementType()
-                .ToElements()
-                .Cast<Autodesk.Revit.DB.FamilyInstance>();
-
-            var matches = familyInstances.Where(x => x.Symbol.Id == familyType.InternalFamilySymbol.Id);
```

Comment:
This leaves the parameter `familyType` not used anymore, now this will return all instances of all families, not only those of the Family Type passed through argument `familyType`. Please integrate back the filtering logic to preserve behavior.

Reply:
Sorry about that. Fixed it.